src/utils/emailController.ts

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
Most of this file is copied from landing-2021: https://github.com/hack-the-6ix/hackthe6ix-landing-2021/blob/main/src/util/email_controller.js

Reply:
Okay, ideally I want to put this behind a lambda but we can do that as an enhancement (which you're free to tackle if you want to learn that)